src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,18 @@
 package core.basesyntax;
 
+import java.util.Arrays;
+
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        final int Number = 3;
+        final int Range = 100;
+        ColorSupplier colorSupplier = new ColorSupplier();
+        Lottery lottery = new Lottery();
+        Ball[] result = new Ball[Number];
+
+        for (int i = 0; i < Number; i++) {
+            result[i] = lottery.getRandomBall(colorSupplier.getRandomColor(), Range);
```

Comment:
The `getRandomBall` method should not accept parameters like `color` and `range`. Instead, it should internally generate these values. This is a critical issue as it does not adhere to the task requirements.